packages/s2-core/src/sheet-type/spread-sheet.ts

```diff
@@ -735,4 +736,26 @@ export abstract class SpreadSheet extends EE {
 
     return w;
   };
+
+  public updateSortMethodMap(
+    nodeId: string,
+    sortMethod: SortMethod,
+    replace = false,
+  ) {
+    const lastSortMethodMap = !replace && this.store.get('sortMethodMap');
```

Comment:
```suggestion
    let lastSortMethodMap: StoreKey['sortMethodMap'] = {};

    if (!replace) {
      lastSortMethodMap = this.store.get('sortMethodMap');
    }
```
这样读起来简单一点，不用去思考 `lastSortMethodMap` 是 `StoreKey['sortMethodMap'] ` 或者 `Boolean` 类型吧

Reply:
我绝对这样反而复杂了吧

Reply:
写起来代码要多一点，我个人觉得维护起来简单点，这个尊重你的看法